cmd/clusterctl/client/cluster/ownergraph.go

```diff
@@ -50,15 +53,20 @@ func nodeToOwnerRef(n *node, attributes ownerReferenceAttributes) metav1.OwnerRe
 
 // GetOwnerGraph returns a graph with all the objects considered by clusterctl move as nodes and the OwnerReference relationship between those objects as edges.
 // NOTE: this data structure is exposed to allow implementation of E2E tests verifying that CAPI can properly rebuild its
-// own owner references; there is no guarantee about the stability of this API.
+// own owner references; there is no guarantee about the stability of this API. Using this test with providers may not require
```

Comment:
nit: Sorry just a bit confused by this sentence: "Using this test with providers may not require a custom implementation of this function, the OwnerGraph it returns."

Please ignore if it's just me

Reply:
Definitely right - I've updated this.